data/transit/route/train.json

```diff
@@ -1725,22 +1737,12 @@
         "route": "train"
       }
     },
-    {
-      "displayName": "TER Midi-Pyrénées",
-      "id": "termidipyrenees-46b9eb",
-      "locationSet": {"include": ["fx"]},
-      "tags": {
-        "network": "TER Midi-Pyrénées",
```

Comment:
It looks like this is still in use (https://taginfo.openstreetmap.org/tags/network=TER%20Midi-Pyr%C3%A9n%C3%A9es). Since this was merged into TER Occitanie, maybe this could be added as one of the [`matchNames`](https://github.com/osmlab/name-suggestion-index/wiki/Contributing#matchnamesmatchtags-optional)

Reply:
Good idea, I will do this when I have some time. Thanks for the suggestion :) !